app/models/user.rb

```diff
@@ -671,7 +667,7 @@ def permissions
         scope: can_access_wfc_senior_matters? ? "*" : [],
       },
       can_access_panels: {
-        scope: panels_with_access,
+        scope: panel_list(self).keys,
```

Comment:
I suppose this `panel_list(self)` comes out of `app/helpers/panel_helper`.
In that case, please move the method somewhere else (for example, as a static/class method directly in `user.rb` because in Rails conventions, `app/helper` is only meant for **UI** helper functions.

Reply:
Reverted this change as this is outdated